cypress/support/swap-page.po.cy.ts

```diff
@@ -0,0 +1,20 @@
+export function getText(selector: string, callback: any) {
+  const text = cy.get(selector).invoke('text')
+  text.then($text => {
+    callback($text)
+  })
+}
+
+export function getList(selector: string, callback: any) {
```

Comment:
rename to getListFavoriteTokens

Reply:
I've updated to "getTokenList" because I also used this function to get list from whitelist.